security/citadel/templates/deployment.yaml

```diff
@@ -55,6 +55,12 @@ spec:
           {{- if .Values.security.trustDomain }}
             - --trust-domain={{ .Values.security.trustDomain }}
           {{- end }}
+          livenessProbe:
+            httpGet:
+              path: /version
+              port: {{ .Values.global.monitoringPort }}
```

Comment:
This is not in globals.yaml.  I believe we decided to hardcode the monitoring port for 1.2.

Reply:
Yes, some of the settings we never tested and likely don't support should be replaced by default values.

We support kustomize for cases user really needs a custom value.